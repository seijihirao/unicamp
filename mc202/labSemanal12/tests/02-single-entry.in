1
0 40000000 -1
0 0
0 40000000
-1 -1
