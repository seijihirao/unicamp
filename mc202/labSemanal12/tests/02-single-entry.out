(0,0) = 0
(0,40000000) = -1
